resources/language/resource.language.es_es/strings.po

```diff
@@ -144,7 +144,7 @@ msgstr "Quitar de la biblioteca"
 
 msgctxt "#30031"
 msgid "General"
-msgstr ""
+msgstr "General"
 
```

Comment:
You can leave blank the destination string (msgstr field) if source string (msgid field) is the same. This is the case.

Reply:
I don't like leave blank the destination string